### PR TITLE
feat(): Differentiate rut from passport

### PR DESCRIPTION
**Pull request by fskarmeta**
Hi, this is associated with this issue https://github.com/platanus/rut-helpers/issues/22 :) .

### CODE REVIEW

File: src/rut-helpers.ts [hunk 1]
```diff
@@ -13,6 +13,10 @@ export function rutValidate(value: string) {
     return false;
   }
 
+  if (/[A-Za-z]/g.test(value.replace(/[kK]/g, ''))) {
+    return false
+  }
+
   const rut: string = rutClean(value);
```
Comment (juanbrujo) [inline src/rut-helpers.ts:20]: I would add a onliner

Comment (fskarmeta) [reply src/rut-helpers.ts:20]: > 

Sure, thought the same, but I figured I better keep the convention already used in the file.

**Comment by gmq** [comment]
Looks perfect!
We talked about it internally and we agreed that it should be fixed. Any chance you can add a test for the new check?

**Comment by fskarmeta** [comment]
> 

Hey @gmq, thats great! Sure I'll be adding a test this weekend.

**Comment by fskarmeta** [comment]
@gmq done :)

**Comment by gmq** [comment]
Awesome, thanks!

**Pull request merged**